Hobbify-Stats/querys.py

```diff
@@ -0,0 +1,91 @@
+import postgres
+
+"""Count instances for today"""
```

Comment:
Docstrings are not compliant with PEP-257. Linters must be run against all source code.